### DISCUSSION TITLE
Use of Curriculum by private school

### DISCUSSION THREAD

**Discussion by MMorro1** [TEALSK12/introduction-to-computer-science#316] [Q&A]
Hi - I would love to use your curriculum.  I teach for an online Catholic Homeschool, and would like to use it to teach computer science, but I want to make sure that I am not violating the "Non-commercial" use clause.  They do pay me, but less than I was making teaching in the public schools :)  

Thank you for your response.

Peggy Morrow

**Comment by DBashirTEALS** [comment]
Hi Peggy,

  Thank you for reaching out to make us aware that you'd like to use the TEALS curriculum.  Since the curriculum is open source and available online, you can definitely utilize it for your class.  Again thanks for making us aware :)

Dr. Bashir

**Reply by MMorro1** [reply]
Thanks so much!

Kind Regards,
Peggy
